helm/cas-ciip-portal/templates/cronJobs/cron-load-testing-data.yaml

```diff
@@ -0,0 +1,74 @@
+{{- if hasSuffix "-dev" .Release.Namespace }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "cas-ciip-portal.fullname" . }}-load-testing-data
+  labels:
+{{ include "cas-ciip-portal.labels" . | indent 4 }}
+  annotations:
+    "helm.sh/hook": "pre-upgrade,pre-install"
+    "helm.sh/hook-delete-policy": "before-hook-creation"
+    "helm.sh/hook-weight": "-5"
```

Comment:
Does this need to be a hook?

Reply:
Probably not? I just copied cron-deploy-data.yaml and changed a couple things

Reply:
This basically replaces the cron-deploy-data step in the dag, so I just copied it and changed the command/name